src/main/java/step3/RacingRule.java

```diff
@@ -0,0 +1,34 @@
+package step3;
+
+import java.util.List;
+import java.util.Random;
+
+public class RacingRule implements Rule {
+    static final int CAN_GO = 4;
+    Random random = new Random();
```

Comment:
이 부분도 `RacingRule`에서만 사용하므로 상수로 선언해도 무방할 것 같습니다 😄 
```suggestion
    private static final Random RANDOM = new Random();
```